src/components/links/external-link.tsx

```diff
@@ -1,18 +1,12 @@
-import React, { Fragment } from 'react'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
-import { IconProp } from '../../utils/iconProp'
-
-export interface ExternalLinkProp {
-  href: string;
-  icon?: IconProp;
-  className?: string
-}
+import React, { Fragment } from 'react'
+import { GeneralLinkProp } from './types'
 
-export interface ExternalLinkTextProp {
+export interface ExternalLinkProps extends GeneralLinkProp {
   text: string;
 }
```

Comment:
since this and `InternalLinkProps` are basicly the same we maybe should extract that to types.ts too?

Reply:
true. done.